conjur_api/http/api.py

```diff
@@ -214,14 +258,18 @@ async def get_variable(self, variable_id: str, version: str = None) -> Optional[
                 'version': version
             }
 
+        api_token = await self.api_token
+        if api_token is None:
+            return
```

Comment:
maybe raise something?
Also Note that this is a breaking change. since the error for Non authenticate call return different error